src/jit/emitxarch.cpp

```diff
@@ -5466,6 +5479,43 @@ void emitter::emitIns_SIMD_R_R_R(instruction ins, emitAttr attr, regNumber reg,
     }
 }
 
+static bool isSSEShift(instruction ins)
```

Comment:
Is `Sse` the appropriate term here, since it also looks to cover AVX instructions, etc?

Reply:
These special encoding shift instructions were introduced by SSE2, and AVX just gives them the VEX-encoding version. There are some AVX/AVX2-only shift instructions that don't belong to this group.